pytorch_blade/pytorch_blade/torch-mlir/lib/Conversion/MhloPasses.cpp

```diff
@@ -42,6 +43,17 @@ void mlir::torch::createDiscTorchBackendToMhloBackendPipeline(
   funcOptions.decompose = false;
   ::mlir::torch::createDiscTorchFunctionToTorchBackendPipeline(pm, funcOptions);
 
+  // Apply pdl pattern match
+  std::string disc_torch_pdl_files;
+  std::string disc_torch_pdll_include_dirs;
+  disc_torch_pdl_files =
```

Comment:
To be consistent with the behavior in tf. Specifically, since pdl is still an experimental feature, it is not suitable to enable it by default.

Reply:
Currently we support two different mode:

1,  pre-defined pdl patterns. These patterns will be compiled with the disc_compiler_main binary.

2,  support read outside pdl patterns at runtime. This mode is also useful since it enables us to extend DISC without re-building it.

We can move a pattern to the pre-defined set once we believe it becomes more stable.